### PR TITLE
config: Add new [program] sections and example scripts to supervisord.conf

### PR DESCRIPTION
**Pull request by elegantcoder**
## Description

I believe the current `supervisord.conf` in the documentation is not suitable for demonstrating a multi-service container. It does not explicitly mention the previously introduced `my_first_script` and `my_second_script`, which reduces its relevance to multi-service container configurations.
To more easily achieve the desired outcome, I have also added examples of `my_first_script` and `my_second_script` to the documentation. 

## Reviews

- [ ] Technical review
- [ ] Editorial review
- [ ] Product review

### CODE REVIEW

**Review by offensive-vk — CHANGES_REQUESTED** [review]
Any other examples other than sleep function?

**Comment by elegantcoder** [comment]
@offensive-vk  For a simple example, I think the sleep function demonstrates well. If you have a better idea, I'd love to hear it.

**Review by offensive-vk — APPROVED** [review]
It seems okay now. Good job

**Comment by offensive-vk** [comment]
Hey @dvdksn , can you please take a look here?
Thank you 🙌

File: content/config/containers/multi-service_container.md [hunk 1]
```diff
@@ -123,20 +142,27 @@ RUN mkdir -p /var/log/supervisor
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY my_first_process my_first_process
 COPY my_second_process my_second_process
+RUN chmod +x my_first_process my_second_process
```
Comment (dvdksn) [inline content/config/containers/multi-service_container.md:145]:
```suggestion
COPY --chmod=755 my_first_process my_second_process .
```

**Review by dvdksn — COMMENTED** [review]
LGTM. One suggestion to the Dockerfile example: we can set chmod and copy both files in one go

**Review by dvdksn — DISMISSED** [review]

**Comment by elegantcoder** [comment]
Hey, I saw the PR got approved! Just checking if you’re planning to merge it soon, or if there's anything else needed?

**Comment by elegantcoder** [comment]
I'm gonna close this pull request. When I first discovered a part where I could contribute to this document, I was happy, but seeing how slow your processing has been, I'm not feeling great about it now.

**Pull request closed without merging**